store/src/java/com/zimbra/soap/SoapEngine.java

```diff
@@ -516,7 +516,8 @@ private Element dispatch(String path, Element envelope, Map<String, Object> cont
                 responseBody = responseProto.soapFault(e);
                 LOG.info("proxy handler exception", e);
             } catch (Throwable e) {
-                responseBody = responseProto.soapFault(ServiceException.FAILURE(e.toString(), e));
+                logFault(e.toString(), null);
```

Comment:
I still think the full exception trace may be needed for diagnosing failures - would go with `logFault(e.toString(), e);`